apicast/t/blacklist.t

```diff
@@ -0,0 +1,68 @@
+BEGIN {
+    $ENV{TEST_NGINX_APICAST_BINARY} ||= 'rover exec apicast';
+    $ENV{APICAST_POLICY_LOAD_PATH} = './policies';
+}
+
+use strict;
+use warnings FATAL => 'all';
+use Test::APIcast::Blackbox 'no_plan';
+
+repeat_each(1);
```

Comment:
I think this one can be run twice. Delete this line so Test::APIcast applies its default?